tests/mail/tests.py

```diff
@@ -1135,6 +1135,35 @@ def test_email_multi_alternatives_content_mimetype_none(self):
         with self.assertRaisesMessage(ValueError, msg):
             email_msg.attach_alternative("<p>content</p>", None)
 
+    def test_body_contains_successful_search(self):
```

Comment:
I would be tempted to combine the `test_body_contains_successful_search`, `test_body_contains_body_contains` and `test_body_contains_alternative_contains` test cases to something like

```python
def test_body_contains(self):
    email_msg = EmailMultiAlternatives()
    email_msg.body = "I am content."
    self.assertIs(email_msg.body_contains("I am"), True)
    self.assertIs(email_msg.body_contains("I am content."), True)

    email_msg.attach_alternative("<p>I am different content.</p>", "text/html")
    self.assertIs(email_msg.body_contains("I am"), True)
    self.assertIs(email_msg.body_contains("I am content."), False)
    self.assertIs(email_msg.body_contains("<p>I am different content.</p>"), False)

Reply:
done 👌